risc0/zkvm/src/receipt.rs

```diff
@@ -13,6 +13,55 @@
 // limitations under the License.
 
 //! Manages the output and cryptographic data for a proven computation
+//!
+//! # Usage
+//! Receipts are typically constructed as the output of the prover
```

Comment:
Not sure what this means. I think we should talk about the journal and how that is a commitment or attestation that verifiers can access.